index.js

```diff
@@ -138,10 +138,14 @@ class VoidBots extends EventEmitter {
       }).then(res => res.json());
     }
 
-   static tokenAvailable() {
+   tokenAvailable() {
      if (!this.token) throw new ReferenceError('No VoidBots token found in this instance');
      return true;
    }
 
 }
-module.exports = VoidBots;
+
+module.exports = {
```

Comment:
```js
module.exports = {
	VoidBots,
	version: require('./package.json').version
};
```
↓
```js
module.exports = VoidBots;
```
This class is already in use by some users, it would be a breaking change to export an object instead of the class itself.